kolibri/core/assets/src/state/actions.js

```diff
@@ -162,7 +162,10 @@ function _channelListState(data) {
  */
 
 function handleError(store, errorString) {
-  store.dispatch('CORE_SET_ERROR', errorString);
+  const authErrorCodes = [401, 403, 407];
+  if (errorCodes.indexOf(errorString.status.code) < 0) {
+    store.dispatch('CORE_SET_ERROR', errorString);
+  }
```

Comment:
putting this logic here might have some unintended consequences for other parts of the app, since it's used in a lot of other places. do you think you can put this only in the spaces needed to fix the original issue (e.g. coach tools)?